proto/firecracker.proto

```diff
@@ -110,5 +110,11 @@ message JailerConfig {
 	// if no value was provided, then /firecracker-containerd will be used as
 	// the default value
     string CgroupPath = 6;
+
+    // DriveExposePolicy is used to configure the method to expose drive files.
+    // "copy" is copying the files to the jail, which is the default behavior.
+    // "bind" is bind-mounting the files on the jail, assuming a caller pre-configures the permissions of
+    // the files appropriately.
+    string DriveExposePolicy = 7;
```

Comment:
Would this make sense as an enum since we don't allow arbitrary strings? Maybe something like:

```proto
enum DrivePolicy {
    COPY = 1;
    BIND = 2;
}
DrivePolicy DriveExposePolicy = 7 [default = BIND];
```

Reply:
I didn't know proto3 can have enums (https://developers.google.com/protocol-buffers/docs/proto3#enum). Yes it does make sense.

Reply:
The default syntax might be proto2, whereas proto3's default value is always zero.